_posts/2015-05-04-The-Role-of-the-API-Designer.md

```diff
@@ -0,0 +1,61 @@
+
+Do not lose hope. There are people in the business who have a vision of where the business needs to be to succeed, and the key levers that make it run (if you are in a company where this is not the case, leave now!). They are Directors, VPs, and Program Managers. They are the folks who have the ear of the C-Level execs.
+
+The key is to tap into the knowledge of these people, carve out boundaries in the silo that can carry that vision, and figure out how to represent it with a set of great APIs.
+
+## Your Customers are Developers
+You might understand the business vision and have a good sense of what business value you need to communicate. If you don’t validate your assumptions as you go with your target market, then you’ll end up building the wrong thing. The startup world has figured this out, but it’s easier for them (though still not easy at all) because they are small enough to have a tightly focused loop between the success of their product and the market they are serving.
+
+An API built to reflect the business vision needs to be developed using the same principles. A tightly-focused feedback loop is needed with the people who will be using the API you’re building. This analogy extends to more than just finding a group of developers to test and iterate the API design. You need to find the right developers, ones who are incentivized to care about that business vision you are trying to articulate in the API. If you only test the API with developers who are building a legacy application, they will only be concerned with legacy needs.
+
+How do you package this up and deliver great APIs? The challenges are significant and a new role is needed to help bring focus and clarity to the API space. API Designers can move the needle on Enterprise API Strategy. By structuring a role that is focused on structurally addressing the two principles above in a targeted way and filling it with people who bring API expertise and a deep understanding of Developer Experience to the table, lasting change is possible and value can be created.
+
+The API Designer focuses on the following:
+
+- Best practices and patterns for modern API design and Developer Experience.
+- Understanding the business vision, value for a well-scoped business domain, and translating it to a great API Design.
+- Acting like a consumer and tightening the consumer feedback loop by facilitating feedback sessions with target API consumers as part of the API design process.
+- Introducing modern API tooling and design practices into the Enterprise.
+- Contributing to best-in-class documentation, tutorials, code libraries and sample code.
+- Working with API Developers to navigate and overcome the constraints of the legacy architecture.
```

Comment:
I don't believe these items in this list need periods. Since they are not sentences, the period doesn't fit.